apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/fragments/CustomPropertyFragment.java

```diff
@@ -0,0 +1,96 @@
+package com.microsoft.appcenter.sasquatch.fragments;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.CheckBox;
+import android.widget.EditText;
+import android.widget.Spinner;
+
+import com.microsoft.appcenter.CustomProperties;
+import com.microsoft.appcenter.sasquatch.R;
+
+public class CustomPropertyFragment extends EditDateTimeFragment {
+
+    public static final int TYPE_CLEAR = 0;
```

Comment:
Blank lines between fields, and within a method blank line must be followed by a comment.